servers/republik/script/prolong/paperInvoice.js

```diff
@@ -29,33 +30,113 @@ const log = printIds
 
 log('Start')
 
+const PROLONG_PACKAGE = '6b8897bf-7ab4-433c-92a1-64fafcd54417'
+const PROLONG_PACAKGEOPTION_ABO = {
+  id: 'c1cb2b16-8f03-4eec-851c-cea4204d7ff7',
+  price: 24000
+}
+const PROLONG_PACAKGEOPTION_BENEFACTOR = {
+  id: '35be2bea-4d3f-44e8-a224-2d17de6d6d51',
+  price: 100000
+}
+
+const createPledgeAndPayments = async ({ transaction, option }, a) =>
+  Promise.map(a, async d => {
+    if (dry) {
+      return { ...d, pledge: {}, pledgeOption: {}, payment: {}, pledgePayments: {} }
+    }
+
+    const pledge = await transaction.public.pledges.insertAndGet({
+      packageId: PROLONG_PACKAGE,
+      userId: d.user.id,
+      status: 'WAITING_FOR_PAYMENT',
+      total: option.price,
+      donation: 0,
+      sendConfirmMail: false
+    })
+
+    const pledgeOption = await transaction.public.pledgeOptions.insertAndGet({
+      templateId: option.id,
+      pledgeId: pledge.id,
+      amount: 1,
+      price: option.price,
+      membershipId: d.membershipId
+    })
+
+    const payment = await transaction.public.payments.insertAndGet({
+      type: 'PLEDGE',
+      method: 'PAYMENTSLIP',
+      paperInvoice: true,
+      total: option.price,
+      status: 'WAITING',
+      dueDate: moment().add(30, 'days')
```

Comment:
use a `now` you define at when the script begins to run so that all payments have the same due date. If you are on it, you could also set the `createdAt` of all these entities to now.